src/botnet.rs

```diff
@@ -0,0 +1,73 @@
+use parse_args::AppConfig;
+use rand::{thread_rng, Rng};
+use utils::{csleep, dprint};
+use yansi::Paint;
+
+pub fn run(appconfig: &AppConfig) {
+    let mut rng = thread_rng();
+    let clusters = {
+        let mut ret = vec![];
+        for _ in 0..rng.gen_range(8, 16 + 1) {
+            ret.push(rng.gen_range(100, 200 + 1));
+        }
+        ret
+    };
+
+    let mut onlines = vec![false; clusters.len()];
+    let size: usize = clusters.iter().sum();
+
+    let mut connected = 0;
+
+    while connected <= size {
+        print!("\rEstablishing connections: {:4}/{:4}", connected, size);
+        connected += 1;
+        csleep((rng.gen_range(0_f64, 1.).powi(50) * 50.) as u64);
+    }
+    println!();
+
+    csleep(300);
+
+    for (i, nodes) in clusters.iter().enumerate() {
+        dprint(format!("  Cluster #{:02} ({:3} nodes)\n", i, nodes), 10);
+        csleep(100);
+    }
+
+    loop {
+        print!("\u{001b}[{}A\n", onlines.len() + 1);
+        for (i, (nodes, online)) in clusters.iter().zip(onlines.iter()).enumerate() {
+            // cannot use bright color because yansi doesn't support it
```

Comment:
I think it does in the form of `.bold()`. Can you check?

Reply:
It does indeed.
I'll fix now.